pkg/fileutils/fileutils.go

```diff
@@ -6,21 +6,43 @@ import (
 )
 
 // Matches returns true if relFilePath matches any of the patterns
+// and isn't excluded by any of the patterns.
```

Comment:
Note to self: This should be "// and isn't excluded by any _subsequent_ pattern."